CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## [0.3.8] - 2024-07-26
```

Comment:
```suggestion
## unreleased
```
We are not sure when we will cut the next release

Reply:
From selfish reasons would be great to have it soon(ish), so I can reduce code base of thisisneat.io 😊